src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENT = 40;
+  const MID_TERM = 3;
+  const LONG_TERM = 7;
+  const MID_TERM_DISC = 20;
+  const LONG_TERM_DISC = 50;
+
+  if (days < MID_TERM) {
+    return DAILY_RENT * days;
+  }
+
+  if (days >= MID_TERM && days < LONG_TERM) {
+    return DAILY_RENT * days - MID_TERM_DISC;
```

Comment:
4. [checklist] Prefer `if` with `return` over if else to simplify later conditions.

( better place long term `if` condition first to prevent repeating second condition here )